docs/contributors/merging/fix-the-changelog.md

```diff
@@ -1,7 +1,7 @@
 (merge-fix-the-changelog)=
 # Fix the changelog
```

Comment:
We may need to adjust the title of this page. "Fix the changelog" is quite a generic statement that I think could apply to many processes.